docs/rules/newline-before-return.md

```diff
@@ -0,0 +1,77 @@
+# Require newline before `return` statement (newline-before-return)
+
+This rule was inspired by [Symfony2's Coding Standards][1]:
```

Comment:
Ah, I was wondering!  Should I move this info to the "Further Reading" section (that came from the generator)?